admin/jobs/river/org_jobs.go

```diff
@@ -173,3 +173,77 @@ func (w *DeleteOrgWorker) Work(ctx context.Context, job *river.Job[DeleteOrgArgs
 
 	return nil
 }
+
+type LogInactiveOrgsArgs struct{}
+
+func (LogInactiveOrgsArgs) Kind() string { return "log_inactive_orgs" }
+
+type LogInactiveOrgsWorker struct {
+	river.WorkerDefaults[LogInactiveOrgsArgs]
+	admin  *admin.Service
+	logger *zap.Logger
+}
+
+func (w *LogInactiveOrgsWorker) Work(ctx context.Context, job *river.Job[LogInactiveOrgsArgs]) error {
+	orgs, err := w.admin.DB.FindInactiveOrganizations(ctx)
+	if err != nil {
+		if errors.Is(err, database.ErrNotFound) {
+			return nil
+		}
+		return fmt.Errorf("failed to find inactive organizations: %w", err)
```

Comment:
No need for `database.ErrNotFound` because it only applies for DB calls that expect exactly one result (e.g. `FindOrganization`, but not `FindOrganizations`).

```suggestion
	if err != nil {
		return fmt.Errorf("failed to find inactive organizations: %w", err)
```